docs/development/guidelines.md

```diff
@@ -56,6 +56,43 @@ Some flag parameters drastically alter the semantics of a function. This can lea
     table.drop("name", axis="columns")
     ```
 
+### Return copies of objects
+
+Modifying objects in-place
+can lead to surprising behaviour
```

Comment:
```suggestion
can lead to surprising behaviours
```

Reply:
I have always thought of "behaviour" as an uncountable noun. I would also, for instance, say that "a program shows unexpected behaviour" (not "*an* unexpected behaviour").
Are you sure about this?